html/migration-to-v2.html

```diff
@@ -5,185 +5,251 @@
+                        Between version 1.4.198 and version 2.0.0 there have been considerable changes, such that a
+                        simple update is
+                        not possible.
+                    </p>
+
+                    <p>
+                        It would have been nice to write some kind of migration tool, or auto-detect the file and
+                        upgrade. Unfortunately, this
+                        is purely a volunteer-run project, so this is just the way it has to be.
+                        There exists a migration tool <a
+                            href="https://github.com/manticore-projects/H2MigrationTool">H2MigrationTool</a> available
+                        in GitHub, but it hasn't been tested by our team. Use at
+                        your own risk.
+                    </p>
+
+                    <h2 id="upgrade">Upgrading</h2>
+
+                    <p>
+                        The official way to upgrade is to do a BACKUP of your existing database <bold>USING YOUR CURRENT
+                            VERSION OF H2.</bold>
+                    </p>
+
+                    <p>
+                        Then create a fresh database <bold>USING THE NEW VERSION OF H2</bold>, then perform a SCRIPT to
+                        load your data.
+                    </p>
+
+                    <h2 id="format">MVStore file format</h2>
+
+                    <p>
+                        The MVStore file format we use (i.e. the default) is still mostly the same, but some subtle
+                        changes have been made
+                        to the undo logs,
+                        for the purposes of improving crash safety and also read/write performance.
+                    </p>
+
+                    <h2 id="datetime">Date / Time / Timestamp</h2>
+
+                    <p>
+                        TODO - Correctness.
+                    </p>
+
+                    <h2 id="general">General</h2>
+
+                    <p>
+                        There are a lot more SQL keywords now. Many SQL statements feature far better support of
+                        SQL-Standard behaviour.
+                        Some old non-standard SQL syntax support has been removed.
+                    </p>
+
+                    <p>
+                        Migrating an old database to the new version works most of the times. However, there are a
+                        couple of important changes in the new version to keep in mind:
+                    </p>
+                    <ul>
+                        <li>
+                            <b>Restricted maximum allowed length limits:</b> there are new length limits for for
+                            non-LOB data types. For example, for the VARBINARY type the maximum allowed size is now
+                            1048576 (1MB)
+                            instead of the previous 16777216 (16MB). Adjust the tables before migration.
```

Comment:
Previous declared limit was 2^31-1, but it wasn't possible to create a value with this length. I have no idea where you found 16MB, but this value is incorrect.